lib/web/assistant_test.go

```diff
@@ -247,8 +247,10 @@ func Test_runAssistError(t *testing.T) {
 	ws, err := s.makeAssistant(t, authPack, conversationID)
 	require.NoError(t, err)
 	t.Cleanup(func() {
-		// Close should yield an error as the server closes the connection
-		require.Error(t, ws.Close())
+		// The websocket has been closed, but the underlying tls connection
```

Comment:
I'd re-phrase that and explain why we are fine with ignoring that error rather than explaining what is happening here.